manifest.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<phar xmlns="https://phar.io/xml/manifest/1.0">
+    <contains name="prophecy/prophecy-phpunit" version="1.0" type="extension">
```

Comment:
hardcoding the version as 1.0 here looks wrong to me

Reply:
The version here should be equal to the version of the package. When we have a release strategy for the phar releases I can have a look at how to implement this properly.

Reply:
well, isn't this PR precisely about implementing that strategy ?